python/ray/train/examples/jax_mnist_example.py

```diff
@@ -0,0 +1,220 @@
+# This example showcases how to use Jax (pmap) with Ray Train.
+# Original code: (without pmap)
+# https://github.com/google/flax/blob/main/examples/mnist/train.py
+import argparse
+from typing import Dict
+from ray.air import session
+
+import functools
+import time
+
+from ray.air.config import ScalingConfig
+from ray.train.jax import JaxTrainer
+import einops
+import numpy as np
+import optax
+import tensorflow_datasets as tfds
+
+
+def get_datasets():
+    import jax
+
+    """Load MNIST train and test datasets into memory."""
+    # shard the dataset
+    def shard_fn(x):
+        # shard the dataset for each node (each process)
+        return einops.rearrange(x, "(d l) ... -> d l ...", d=jax.process_count())[
+            jax.process_index()
+        ]
```

Comment:
nit: split this into two lines
```suggestion
        # shard the dataset for each node (each process)
        shards = einops.rearrange(x, "(d l) ... -> d l ...", d=jax.process_count())
        return shards[jax.process_index()]
```

Reply:
smart